### DISCUSSION TITLE
Thin propositions should be moved to doc

### DISCUSSION THREAD

**Discussion by antoinerrr** [topolvm/topolvm#982] [Q&A] [answered]
Hi, I'm wondering if docs/proposals/thin-volumes.md and all thin related proposals should be moved to the root doc folder, as I think they are all implemented ?
Thanks

**Comment by pluser** [comment, marked as answer]
Hello. Thank you for your suggestion.
Proposals are different from product usage and may sometimes diverge from implementation. I believe it is appropriate to keep the proposals and add the usage related to thin volumes to the docs/.
